### ISSUE TITLE
Node suddenly in state "not ready". Re-install does not help. 

### ISSUE THREAD

**Issue by AVSuni** [canonical/microk8s#3088]
I have one node failing in my cluster as of last night. The state was showing as `not ready` and all pods were `terminating`. `microk8s status` gives `microk8s is not running`.  `microk8s leave` had no effect  so I had to do `microk8s remove-node --force`.  I can reinstall microk8s on the node just fine, but the second I try joining it to the cluster microk8s dies on the node. Any ideas? Something similar has happened to me before that led me having to reinstall the entire cluster. I'm hoping I won't have to do that again.  Thanks for your help.
[inspection-report-20220424_181159.tar.gz](https://github.com/canonical/microk8s/files/8549987/inspection-report-20220424_181159.tar.gz)

**Comment by balchua** [comment]
This node's dqlite isn't starting.  I looked at the error logs, it seems to be similar to this.
https://github.com/canonical/microk8s/issues/2531

Can you check the content of `cluster.yaml`?

And did you change the hostname to `devbox5`?

**Comment by AVSuni** [comment]
Ok, it works with a new IP address. I kept the name as the original one. This is now the second time I encounter this. Any chance that dqlite becomes more easily corrupt in systems with non-ECC ram? I'm currently moving my cluster to ECC

**Comment by balchua** [comment]
Hi @AVSuni i didn't recall nor experienced non ECC being an issue.  Was the node rebooted or restarted ungracefully?

**Comment by AVSuni** [comment]
No, actually that node was untouched and up. It was a "datastore standby node" at the time. But I did notice one of the master nodes becoming unresponsive and had to reboot that node.  I'm not 100% sure if the standby node started its childish behaviour at the same time.  I'm closing this issue for now, but this is the second time that I've witnessed a node recover only once 
1) it's been removed from the cluster
2) its IP address has been changed
